webapp/security/views.py

```diff
@@ -333,5 +342,208 @@ def cve(cve_id):
 # ===
 
 
-def api_create_cve():
-    pass
+def create_cve():
+    """
+    Receives a POST request from load_cve.py
+    Parses the object and bulk inserts with add_all()
+    @params json: the body of the request
+    """
+
+    data = flask.request.get_json()
+    response = flask.jsonify({"message": "Unable to get body"}), 400
+    packages = []
+    references = []
+    bugs = []
+
+    # Check if CVE exists by candidate
+    if db_session.query(CVE).filter(CVE.id == data["id"]).count() > 0:
+        response = flask.jsonify({"message": "CVE already exists"}), 400
+        return response
+
+    # Packages
+    # Check if there are packages before mapping
+    if "packages" in data and len(data["packages"]) > 0:
+        for pkg in data["packages"]:
+            releases = []
+            for rel in pkg["releases"]:
+                release = PackageReleaseStatus(
+                    name=rel["name"],
+                    status=rel["status"],
+                    status_description=rel["status_description"],
+                )
+            releases.append(release)
+            package = Package(
+                name=pkg["name"],
+                source=pkg["source"],
+                ubuntu=pkg["ubuntu"],
+                debian=pkg["debian"],
+                releases=releases,
+            )
```

Comment:
Similarly, how about:

```suggestion
            packages.append(
                Package(
                    name=pkg["name"],
                    source=pkg["source"],
                    ubuntu=pkg["ubuntu"],
                    debian=pkg["debian"],
                    releases=releases,
                )
            )
```